harness/tests/tensorboard/test_s3.py

```diff
@@ -11,6 +11,29 @@
 
 BASE_PATH = pathlib.Path(__file__).resolve().parent.joinpath("fixtures")
 
+test_configs = [
```

Comment:
rather than create 3x as many copies of the default_conf, can we just add `@pytest.mark.parameterize` a `prefix` parameter into the `test_s3_build()` test, do something like:
```python
conf = dict(**default_conf)
if prefix is not None:
    conf["prefix"] = prefix
```